scripts/arm32_ci_script.sh

```diff
@@ -203,9 +264,17 @@ do
             exit_with_error "--buildConfig can be only Debug or Release" true
         fi
         ;;
+    --arm)
+        __ARMRootfsImageBase="rootfs-u1404.ext4"
```

Comment:
@gkhanna79 Not necessary, but it is not required for build. I just leave this code to keep consistency the script and this variable can be used when we enable testing using ARM emulator.